doc/source/developer_guide/dg_forcing.rst

```diff
@@ -0,0 +1,216 @@
+
+The forcing is read and interpolated in subroutine **JRA55_data**.  In particular,
+air temperature (airtmp), east and north wind speed (wndewd and wndnwd), 
+specific humidity (spchmd), incoming short and longwave radiation (glbrad and dswsfc),
+and precipitation (ttlpcp) are read from the input files.   The JRA55 reanalysis is 
+run with updated initial conditions every 6 hours and output is written every 3 hours.
+The four state fields (air temperature, winds, specific humidity)
+are instantaneous data, while the three flux fields (radition, precipitation) are 3
+hour averages.  In the JRA55 forcing files provided by the Consortium, the time 
+defined for 3 hour average fields is shifted 3 hours to the start time of the 3 
+hour interval.  **NOTE that this is different
+from the implementation on the original JRA55 files and also different from how models
+normally define time on an accumulated/averaged field**.  This is all shown 
+schematically in Figure :ref:`fig-jra55data`.  
+
+The state fields are linearly 
+time interpolated between input timestamps 
+while the flux fields are read and held constant during each 3 hour model period.
+The forcing frequency is hardwired to 3 hours in the implementation,
+and the record number is computed based on the time of the current model year.
+Time interpolation coefficients are computed in the **JRA55_data** subroutine.
+
+The forcing data is converted to model inputs in the subroutine **prepare_forcing**
+called in **get_forcing_atmo**.  To clarify, the JRA55 input data includes
+
+- uatm    = model grid i-direction wind velocity component (m/s)
+- vatm    = model grid j-direction wind velocity component (m/s)
+- Tair    = air temperature  (K)
+- Qa      = specific humidity (kg/kg)
+- flw     = incoming longwave radiation (W/m^2)
+- fsw     = incoming shortwave radiation (W/m^2)
+- fsnow   = snowfall rate (kg/m^2 s)
+
+and model forcing inputs are derived from those fields and the defaults.
+
+Because the input files are on the gregorian time axis, the model can run with the regular
+365 day (noleap) calendar, but in that case, the Feb 29 input data will be used on 
+March 1, and all data
+after March 1 will be shifted one day.  December 31 in leap years will be skipped when
+running with a CICE calendar with no leap days.
+
+
+.. _NCARforcing:
+
+NCAR Atmosphere Forcing
+-------------------------
+
+The NCAR atmospheric forcing was used in earlier standalone runs on the gx3 grid, and the
+Consortium continues to do some limited testing with this forcing dataset.
+Monthly average data for fsw, cldf, fsnow are read.  6-hourly data for
+Tair, uatm, vatm, rhoa, and Qa are also read.
+Users are encouraged to switch to the JRA55 (see :ref:`JRA55forcing`) dataset.  This
+atmosphere forcing dataset may be deprecated in the future.
+
+
+.. _LYqforcing:
+
+LYq Atmosphere Forcing
+-------------------------
+
+The LYq (Large and Yeager :cite:`Large09`) forcing was used in earlier standalone 
```

Comment:
The Hunke and Holland 2007 paper should be cited here, since LYq is partly Large and Yeager's CORE II forcing (u, v, T, Q) and partly AOMIP (radiation, precip).

Reply:
I added Hunke07 and updated the reference.